tests/fast-integration/utils/index.js

```diff
@@ -1191,8 +1196,12 @@ function genRandom(len) {
   return res;
 }
 
+function isEnvSet(key) {
```

Comment:
Function name suggest boolean value, but you get string. I don't think that adding this function helps.